Documentation/BuildInstructionsOther.md

```diff
@@ -48,11 +48,12 @@ apk add coreutils
 apk add build-base
 
 # qemu
-apk add qemu qemu-system-i386 qemu-img qemu-ui-gtk
+apk add qemu qemu-system-x86_64 qemu-img qemu-ui-gtk qemu-audio-pa
 
 # build tools (samurai is a drop-in replacement for ninja)
 apk add cmake e2fsprogs grub-bios samurai mpc1-dev mpfr-dev gmp-dev ccache rsync texinfo
 ```
+Optional: `fuse2fs` for [building images without root](https://github.com/SerenityOS/serenity/pull/11224).
```

Comment:
Can you add a newline after the codeblock / before this line? Also, I'm not sure if linking to a PR is relevant here.

Reply:
I linked the PR because you need fuse2fs if you don't want to build as root (same as some other distro).

Reply:
> I linked the PR because you need fuse2fs if you don't want to build as root (same as some other distro).

Yes, but if you install `fuse2fs` that will just work out of the box - so no additional link required. If there's something in that PR that needs documenting, feel free to copy that over instead!

> I didn't add a newline because in the other paragraphs it's not present (see fedora or freebsd). Should I add one there too?

Ah, you copied it exactly from the block below and changed the package name to `fuse2fs`. That explains why it looks like that :^)

> This it's the first PR that I make, should I make a second commit and push or amend the one that I already made?

Let's not change it, since you copied it. Usually you would amend commits, yeah.